sql/npc_list.sql

```diff
@@ -27716,7 +27716,7 @@ INSERT INTO `npc_list` VALUES (17736072,'Tales_Beginning','Tales\' Beginning',11
 -- Bastok Markets (Zone 235)
 -- ------------------------------------------------------------
 
-INSERT INTO `npc_list` VALUES (17739777,'NPC[1]','NPC',171,-261.242,-12.136,80.643,0,50,50,50,0,0,0,6,0x0000620000000000000000000000000000000000,32,NULL,0);
+INSERT INTO `npc_list` VALUES (17739777,'NPC[1]','NPC',171,-261.242,-12.136,80.643,0,50,50,50,0,0,3,6,0x0000620000000000000000000000000000000000,32,NULL,0);
```

Comment:
This fix is incorrect.
Just because the npc should not be visible, doesnt automatically mean the NPC status should be "Invisible"

For this kind of stuff, a capture is needed. One can't just make up this values.

Once you look at the sql entry, the issue is obvious. It has an extra speed value.

Reply:
Fixed.